components/widget-welcome/index.jsx

```diff
@@ -26,36 +26,36 @@ var Widget_Welcome = React.createClass( {
 							<span>Important service updates</span>
 						</li>
 						<li>
-							<a href="https://vip.wordpress.com/documentation/vip-go/" target="_blank">VIP Go Documentation</a>
-							<span>Coding for WordPress.com VIP Go</span>
-						</li>
-						<li>
-							<a href="https://vip.wordpress.com/plugins/" target="_blank">VIP Plugins</a>
-							<span>Approved VIP plugins</span>
+							<a href="https://wpvip.com/documentation/" target="_blank">VIP Documentation</a>
+							<span>Launching and developing with VIP</span>
 						</li>
 						<li>
 							<a href="https://wordpressvip.zendesk.com/" target="_blank">VIP Support Portal</a>
 							<span>Your organization’s tickets</span>
 						</li>
+						<li>
+							<a href="https://wpvip.com/documentation/general-ticket-guidelines/" target="_blank">Ticket guidelines</a>
+							<span>How to open the perfect ticket</span>
+						</li>
 					</ul>
 				</div>
 
 				<div className="widget__col-2">
 					<ul className="widget__list">
 						<li>
-							<a href="https://vip.wordpress.com/documentation/vip-go/launch-checklist/" target="_blank">Launch Checklist</a>
+							<a href="https://wpvip.com/documentation/launching-with-vip/" target="_blank">Guidebook: Launching with VIP</a>
```

Comment:
```suggestion
							<a href="https://wpvip.com/documentation/launching-with-vip/welcome-to-your-vip-launch/" target="_blank">Guidebook: Launching with VIP</a>
```

The linked url actually redirects to this, so we could skip a step here, what do you think?

Reply:
Either one works!